inspirehep/dojson/journals/fields/bd1xx.py

```diff
@@ -108,10 +108,3 @@ def title_variants(self, key, value):
     return {
         'title': value.get('a'),
     }
-
-
-@journals.over('name_variants', '^730..')
-@utils.for_each_value
-def name_variants(self, key, value):
-    """Variants of the name."""
-    return value.get('a')
```

Comment:
`730__a` is already consumed by rule `title_variants`, which populates the field `title_variants`, which is in the schema, while `name_variants` isn't there: https://github.com/inspirehep/inspire-next/blob/303444d0e344d3a4c750d82c45c6094a6f224e05/inspirehep/modules/records/jsonschemas/records/journals.json